tensorflow/tools/ci_build/builds/pip.sh

```diff
@@ -96,8 +101,7 @@ source tools/python_bin_path.sh
 
 # Assume: PYTHON_BIN_PATH is exported by the script above
 if [[ -z "$PYTHON_BIN_PATH" ]]; then
-  echo "PYTHON_BIN_PATH was not provided. Did you run configure?"
-  exit 1
+  die "PYTHON_BIN_PATH was not provided. Did you run configure?"
```

Comment:
I'm fine with the rest of the change though. They reduce the code in size a little and achieve the same purpose.